esp-wifi/MIGRATING-0.11.md

```diff
@@ -17,3 +17,7 @@ configurations to match the new format.
 +ESP_WIFI_CONFIG_STATIC_RX_BUF_NUM = "32"
 +ESP_WIFI_CONFIG_DYNAMIC_RX_BUF_NUM = "16"
 ```
+
+## `csi_enabled` option converted to feature
+
+As part of limiting public API changes due to config options, the `csi_enabled` option has been changed to feature. The feature must now be activated in `esp-wifi` crate to activate the corresponding functionality. 
```

Comment:
Not sure the reasoning of the change belongs here - but we have similar phrasing in other places

Reply:
The guide needs to spell out WHICH configuration option needs to be enabled, otherwise it's just a meaningless duplication of the changelog entry.